pipenv/utils/virtualenv.py

```diff
@@ -237,6 +238,22 @@ def abort(msg=""):
     # Find out which python is desired.
     if not python:
         python = project.required_python_version
+        if python:
+            range_pattern = r"^[<>]=?|!="
+            if re.search(range_pattern, python):
+                click.echo(
+                    "{}: Python version range specifier '{}' is not supported. {}".format(
+                        click.style("Error", fg="red", bold=True),
+                        click.style(python, fg="cyan"),
+                        click.style(
+                            "Please use an absolute version number or specify the path to the Python executable on Pipfile.",
+                            fg="yellow",
+                        ),
+                    ),
+                    err=True,
+                )
+                abort()
```

Comment:
Maybe `abort` is fine, not sure how it behaves different then the other runtime errors we raise--just calling out this is an uncommon pattern in the project so deserves additional thought to standardize on a best approach.

Reply:
@matteius I'll think about it more about how to handle the runtime errors in another ways.

Reply:
How about `sys.exit(1)`?   That is what I am using in my PR to clean up and not render a stack error trace when resolver fails.